src/tags/object/Paragraphs/HtxParagraphs.js

```diff
@@ -471,7 +489,16 @@ class HtxParagraphsView extends Component {
   }
 
   _handleScrollRoot() {
-    this._disposeTimeout();
+    if (this.activeRef.current) {
+      const { top, bottom, height } = this.activeRef.current.getBoundingClientRect();
+      const { offsetHeight, offsetTop } = this.myRef.current;
+      const offset = offsetTop + 95;
```

Comment:
What is 95 here? If this is a constant we should store it as such at module level with a descriptive enough name.